doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -5,37 +5,37 @@ Introduction
+standard-independent Kubernetes reference architecture that is not tied to any
+specific offering or distribution. No vendor-specific enhancements are required,
+in order to achieve conformance to the principles of Anuket specifications.
+Conformance is achieved by using upstream components or features that are
+developed by the open source community. This allows operators to have a common
+Kubernetes-based architecture that supports any conformant VNF or CNF deployed on
+it to operate as expected. The purpose of this chapter is to outline all the
+components required to provide Kubernetes in a consistent and reliable way. The
+specification of how to use these components is detailed in
+Chapter 04 :ref:`chapters/chapter04:component level architecture`.
+
+Kubernetes is already a well-documented and widely deployed open-source project
 managed by the Cloud Native Computing Foundation (CNCF). Full documentation of
 the Kubernetes code and project can be found at
-`https://kubernetes.io/docs/home/ <https://kubernetes.io/docs/home/>`__. The
-following chapters will only describe the specific features required by the Anuket
-Reference Architecture, and how they would be expected to be implemented. For
-any information related to standard Kubernetes features and capabilities, refer
-back to the standard Kubernetes documentation.
-
-While this reference architecture provides options for pluggable components such
-as service mesh and other plugins that might be used, the focus of the
-reference architecture is on the abstracted interfaces and features that are
-required for telco type workload management and execution.
-
-Chapter 5 of the Reference Model (RM) describes the
-hardware and software profiles that are
-descriptions of the capabilities and features that the Cloud Infrastructure
-provide to the workloads. The NFVI Software Profile figure below
-depicts a high level view of the software profile features that apply to each
-instance profile (Basic and High Performance). For more information on the
-instance profiles please refer to :ref:`ref_model:chapters/chapter04:profiles`.
+`https://kubernetes.io/docs/home/ <https://kubernetes.io/docs/home/>`__.
+The following chapters only describe the specific features required by the
+Anuket Reference Architecture, and how they would be expected to be implemented.
+For information related to standard Kubernetes features and capabilities, see
+the standard Kubernetes documentation.
+
+While this reference architecture provides options for pluggable components,
+such as service mesh and other plugins, the focus of the reference architecture
+is on the abstracted interfaces and features that are required for Telco-type
+workload management and execution.
+
+Chapter 5 of the Reference Model (RM) describes the hardware and software profiles
+that are descriptions of the capabilities and features that the Cloud Infrastructure
+provide to the workloads. As of v2.0, Figure 5-3 in the RM (also shown below)
```

Comment:
```suggestion
provides to the workloads. Figure 5-3 in the RM (reproduced below)
```

Reply:
I would be inclined to keep "also shown" as is.